README.md

```diff
@@ -95,6 +95,23 @@ return zecko.cartClient.addToCart({
 return zecko.cartClient.completeCartById('YOUR_CART_ID', {
   // `YOUR_PAYMENT_METHOD` can be one of `COD or PREPAID`
   paymentMethod: 'YOUR_PAYMENT_METHOD',
+  discountCode: 'YOUR_DISCOUNT_COUPON_CODE',
+  shippingAddress: {
+    // Typically for the street address or PO Box number
+    address1: 'YOUR_CUSTOMER_ADDRESS',
+    // Optional Field, Typically for the number of the apartment, suite, or unit
```

Comment:
These comments should be on the same line as the field itself. Otherwise it can get confusing if you're referring to top one or bottom one